File: 1/java/p1.java
class P1 {
  int number;
  public P1(int number) {
    this.number = number;
  }

  public int result() {
    int sum = 0;
    for(int i=1; i < this.number; i++){
      if(i % 3 == 0 || i % 5 == 0) {
        sum += i;
      }
    }
    return sum;
  }
}

File: 1/node/p1.js
var p1 = function (number) {
  var result = 0
  for( var i=1;i<number; i++) {
    if (i % 3 === 0 || i % 5 === 0 ){
      result += i
    }
  }
  return result
} 

module.exports = p1
